app/pages/schedule.vue

```diff
@@ -153,6 +103,13 @@ export default {
       this.$refs[dayindex][0].classList.add('block')
       this.$refs[dayindex][0].classList.remove('hidden')
       this.selectedDay = dayindex
+    },
+    checkNowPlaying () {
+      this.$axios.get(this.streamServer).then((response) => {
+        this.nowPlaying = response.data
+      }).catch((error) => {
+        console.log(error)
```

Comment:
Maybe something else instead of console log? For example we can create custom errors for Sentry like this:
 - in vue components: `this.$sentry.captureException(new Error('example'))`
 - in async: `catch (error) {
    $sentry.captureException(error) }`
 - in middleware: `process.$sentry`
 
 See more here: https://sentry.nuxtjs.org/guide/usage

Reply:
wow, yess, we can, definitely! but I think it is better to do it globally, for all the error messages, I'll open a new issue with this